tests/test_app.py

```diff
@@ -188,7 +188,27 @@ def test_packages_empty(testapp):
     assert len(resp.html("a")) == 0
 
 
-def test_health(testapp):
+def test_health_default_endpoint(testapp):
+    resp = testapp.get("/health")
+    assert resp.status_int == 200
+    assert "Ok" in resp
+
+
+def test_health_customized_endpoint(root):
+    from pypiserver import app
+
+    testapp = webtest.TestApp(
+        app(root=root.strpath, health_endpoint="/healthz")
+    )
```

Comment:
```suggestion
    _app = app(root=root.strpath, health_endpoint="/healthz")
    testapp = webtest.TestApp(_app)
```

Just to keep the lines similar to other test setups :)